packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -247,3 +247,16 @@ def get_log_record_extra(*, user_id: int | str | None = None) -> LogExtra | None
         assert int(user_id) > 0  # nosec
         extra["log_uid"] = f"{user_id}"
     return extra or None
+
+
+def guess_message_log_level(message: str) -> int:
+    lower_case_message = message.lower().strip()
+    if lower_case_message.startswith(
+        ("error:", "err:", "error ", "err ", "[error]", "[err]")
+    ):
```

Comment:
If you want to improve in the future the guess, you can even use Levenshtein distance algorithm (there are python packages that do that for you)

Reply:
link?

Reply:
thanks!